### ISSUE TITLE
Update jQuery to 3.5.0

### ISSUE THREAD

**Issue by BrookeDot** [PrivateBin/PrivateBin#608]
I see there is some discussion around removing jQuery in #198 however for now it remains part of the project. On April 10th 2020 the jQuery team [released 3.5.0](https://blog.jquery.com/2020/04/10/jquery-3-5-0-released/) which contains a security fix and a few other changes. 

It would be great to get jQuery removed (or updated) before the next release. I'd be happy to submit a PR for updating if I can get more information on the best way to generate the Subresource Integrity (SRI) hash.

**Comment by Haocen** [comment]
You can easily regenerate SRI by running the following code snippet in your browser dev console:

```
fetch('https://privatebin.net/js/privatebin.js').then(r => r.blob()).then(b => b.arrayBuffer()).then(t => crypto.subtle.digest('SHA-512', t)).then(hab => console.log(btoa(String.fromCharCode(...new Uint8Array(hab)))))
```

Of course, replace `privatebin.net` with your own deploy.

**Comment by rugk** [comment]
Off-topic, but nice tip, @Haocen. Would you mind to [add this here to the contributing guide](https://github.com/PrivateBin/PrivateBin/wiki/Development#subresource-integrity-for-javascript-resources)?

BTW: @BrookeDot Sorry your initial question got lost. As my previous link explains, one (other) way to regenerate the SRI hashes is to run the PHP unit tests.

**Comment by elrido** [comment]
Solved in #880
